packages/platform-test-suite/test/functional/platform/Identity.spec.js

```diff
@@ -68,8 +71,8 @@ describe('Platform', () => {
 
       const {
         identityCreateTransition: invalidIdentityCreateTransition,
-      } = await createIdentityCreateTransition(
-        client.platform, assetLockProof, privateKey,
+      } = await client.platform.internal.createIdentityCreateTransition(
```

Comment:
As I understood internal functions are supposed to be internal )). Maybe we should make them public since we are using them from outside in many places? They could be under `client.platform.identities.util...` namespace for example.

Reply:
I made them internal on purpose to reduce the liability of the SDK public API scope. 
The fewer API functions we expose, the less is the scope of backward compatibility checks for us :)

Reply:
Agreed to move util functions to the `identities.utils` scope. Done